<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWidget</class>
 <widget class="QWidget" name="GameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item row="0" column="8">
    <widget class="QPushButton" name="accountButton">
     <property name="text">
      <string>Account Name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="9">
    <widget class="QPushButton" name="logoutButton">
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="4" column="1">
      <widget class="QLabel" name="drawsCounter">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>10000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="lossesCounter">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>69</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="winsLabel">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wins:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="winsCounter">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>420</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="drawsLabel">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Draws:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lossesLabel">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Losses:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="5">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tic-Tac-Toe Infinity 2.0</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <layout class="QGridLayout" name="gameBoard">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="2" column="1">
      <widget class="ClickableLabel" name="bottomMiddle">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>130</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-left: 5px solid black; border-right: 5px solid black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="ClickableLabel" name="middleRight">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>130</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-top: 5px solid black; border-bottom: 5px solid black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ClickableLabel" name="middle">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>130</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 5px solid black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="ClickableLabel" name="middleLeft">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>130</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-top: 5px solid black; border-bottom: 5px solid black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ClickableLabel" name="topMiddle">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>130</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-left: 5px solid black; border-right: 5px solid black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="ClickableLabel" name="bottomLeft">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>130</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="ClickableLabel" name="topRight">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>130</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="ClickableLabel" name="bottomRight">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>130</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="ClickableLabel" name="topLeft">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>130</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Stats</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="5">
    <widget class="QLabel" name="turnLabel">
     <property name="font">
      <font>
       <pointsize>26</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">margin-bottom:20px;</string>
     </property>
     <property name="text">
      <string>It's your turn</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="8">
    <widget class="QPushButton" name="newGameButton">
     <property name="text">
      <string>New Game</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>ClickableLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
